### ISSUE TITLE
Instant Crash with limited barrel

### ISSUE THREAD

**Issue by Darkblaster14** [P3pp3rF1y/SophisticatedStorage#120]
Hello there,

I have an issue with this mod playing on a server. I am using version "sophisticatedstorage-1.19.2-0.6.16.276.jar" of the mod and have no issue before crafting an limited barrel (don't remember the exact name of the item sorry) and uprgade it to gold. It appear that when I tried to put stacks of deepslate in it (I saw 1024 number diplay) my game instant crash. I can't log back it keep crashing and crashing again. I could unistall your mod as a fix but I have a lot of chest comming from it and I don't know what will happend to the stuff inside (don't want to lose it)

The servezr isn't crashing only my game.
 

Here is the crash error : 

The game crashed whilst tesselating block model
Error: java.lang.NoSuchMethodError: 'boolean net.minecraft.client.renderer.block.model.BakedQuad.hasAmbientOcclusion()'

and the crash log :

---- Minecraft Crash Report ----
// This doesn't make any sense!

Time: 2023-01-03 20:10:16
Description: Tesselating block model

java.lang.NoSuchMethodError: 'boolean net.minecraft.client.renderer.block.model.BakedQuad.hasAmbientOcclusion()'
	at net.minecraftforge.client.model.IQuadTransformer.copy(IQuadTransformer.java:95) ~[forge-1.19.2-43.2.0-universal.jar%23178!/:?] {re:classloading}
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197) ~[?:?] {}
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625) ~[?:?] {}
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?] {}
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?] {}
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575) ~[?:?] {}
	at java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260) ~[?:?] {}
	at java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616) ~[?:?] {}
	at java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:622) ~[?:?] {}
	at java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:627) ~[?:?] {}
	at net.minecraftforge.client.model.IQuadTransformer.process(IQuadTransformer.java:48) ~[forge-1.19.2-43.2.0-universal.jar%23178!/:?] {re:classloading}
	at net.p3pp3rf1y.sophisticatedstorage.client.render.BarrelBakedModelBase.addRenderedItemSide(BarrelBakedModelBase.java:303) ~[sophisticatedstorage-1.19.2-0.6.16.276.jar%23171!/:1.19.2-0.6.16.276] {re:classloading}
	at net.p3pp3rf1y.sophisticatedstorage.client.render.BarrelBakedModelBase.addDisplayItemQuads(BarrelBakedModelBase.java:290) ~[sophisticatedstorage-1.19.2-0.6.16.276.jar%23171!/:1.19.2-0.6.16.276] {re:classloading}
	at net.p3pp3rf1y.sophisticatedstorage.client.render.BarrelBakedModelBase.getQuads(BarrelBakedModelBase.java:193) ~[sophisticatedstorage-1.19.2-0.6.16.276.jar%23171!/:1.19.2-0.6.16.276] {re:classloading}
	at net.minecraft.client.renderer.block.ModelBlockRenderer.renderModelFlat(ModelBlockRenderer.java:236) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.block.ModelBlockRenderer.tesselateBlock(ModelBlockRenderer.java:114) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.block.BlockRenderDispatcher.renderBatched(BlockRenderDispatcher.java:90) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$RenderChunk$RebuildTask.m_234467_(ChunkRenderDispatcher.java:1065) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$RenderChunk$RebuildTask.m_5869_(ChunkRenderDispatcher.java:908) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher.lambda$runTask$0(ChunkRenderDispatcher.java:195) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {re:computing_frames}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {re:computing_frames}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {}

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraftforge.client.model.IQuadTransformer.copy(IQuadTransformer.java:95) ~[forge-1.19.2-43.2.0-universal.jar%23178!/:?] {re:classloading}
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197) ~[?:?] {}
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625) ~[?:?] {}
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?] {}
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?] {}
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575) ~[?:?] {}
	at java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260) ~[?:?] {}
	at java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616) ~[?:?] {}
-- Block model being tesselated --
Details:
	Block: Block{sophisticatedstorage:limited_gold_barrel_1}[facing=south,ticking=false,vertical_facing=no]
	Block location: World: (-874,70,-668), Section: (at 6,6,4 in -55,4,-42; chunk contains blocks -880,-64,-672 to -865,319,-657), Region: (-2,-2; contains chunks -64,-64 to -33,-33, blocks -1024,-64,-1024 to -513,319,-513)
	Using AO: false
Stacktrace:
	at net.minecraft.client.renderer.block.ModelBlockRenderer.tesselateBlock(ModelBlockRenderer.java:114) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.block.BlockRenderDispatcher.renderBatched(BlockRenderDispatcher.java:90) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$RenderChunk$RebuildTask.m_234467_(ChunkRenderDispatcher.java:1065) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$RenderChunk$RebuildTask.m_5869_(ChunkRenderDispatcher.java:908) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher.lambda$runTask$0(ChunkRenderDispatcher.java:195) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {re:computing_frames}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {re:computing_frames}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {}

-- Block being tesselated --
Details:
	Block: Block{sophisticatedstorage:limited_gold_barrel_1}[facing=south,ticking=false,vertical_facing=no]
	Block location: World: (-874,70,-668), Section: (at 6,6,4 in -55,4,-42; chunk contains blocks -880,-64,-672 to -865,319,-657), Region: (-2,-2; contains chunks -64,-64 to -33,-33, blocks -1024,-64,-1024 to -513,319,-513)
Stacktrace:
	at net.minecraft.client.renderer.block.BlockRenderDispatcher.renderBatched(BlockRenderDispatcher.java:90) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$RenderChunk$RebuildTask.m_234467_(ChunkRenderDispatcher.java:1065) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$RenderChunk$RebuildTask.m_5869_(ChunkRenderDispatcher.java:908) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher.lambda$runTask$0(ChunkRenderDispatcher.java:195) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,xf:OptiFine:default}
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {re:computing_frames}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {re:computing_frames}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {}

-- Affected level --
Details:
	All players: 1 total; [LocalPlayer['Darkouille153'/264, l='ClientLevel', x=-872.10, y=70.00, z=-665.15]]
	Chunk stats: 289, 75
	Level dimension: minecraft:overworld
	Level spawn location: World: (16,70,-432), Section: (at 0,6,0 in 1,4,-27; chunk contains blocks 16,-64,-432 to 31,319,-417), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,-64,-512 to 511,319,-1)
	Level time: 1321402 game time, 1963925 day time
	Server brand: forge
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.ClientLevel.m_6026_(ClientLevel.java:580) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:mixin,xf:OptiFine:default,re:classloading,xf:OptiFine:default,pl:mixin:APP:architectury.mixins.json:MixinClientLevel,pl:mixin:A}
	at net.minecraft.client.Minecraft.m_91354_(Minecraft.java:2278) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_231462_(Minecraft.java:800) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:690) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.m_239872_(Main.java:212) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:51) ~[client-1.19.2-20220805.130853-srg.jar%23173!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27) ~[fmlloader-1.19.2-43.2.0.jar%2395!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) [modlauncher-10.0.8.jar%2382!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-10.0.8.jar%2382!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-10.0.8.jar%2382!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-10.0.8.jar%2382!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-10.0.8.jar%2382!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-10.0.8.jar%2382!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-10.0.8.jar%2382!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) [bootstraplauncher-1.1.2.jar:?] {}

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Mod Resources, Default, FreshAnimations_v1.7.zip, VisibleOres2.8.zip

-- System Details --
Details:
	Minecraft Version: 1.19.2
	Minecraft Version ID: 1.19.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 288469264 bytes (275 MiB) / 838860800 bytes (800 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 4
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i3-1005G1 CPU @ 1.20GHz
	Identifier: Intel64 Family 6 Model 126 Stepping 5
	Microarchitecture: Ice Lake (Client)
	Frequency (GHz): 1.19
	Number of physical packages: 1
	Number of physical CPUs: 2
	Number of logical CPUs: 4
	Graphics card #0 name: Intel(R) UHD Graphics
	Graphics card #0 vendor: Intel Corporation (0x8086)
	Graphics card #0 VRAM (MB): 1024.00
	Graphics card #0 deviceId: 0x8a56
	Graphics card #0 versionInfo: DriverVersion=30.0.100.9805
	Memory slot #0 capacity (MB): 4096.00
	Memory slot #0 clockSpeed (GHz): 2.67
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 4096.00
	Memory slot #1 clockSpeed (GHz): 2.67
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 22582.46
	Virtual memory used (MB): 14625.53
	Swap memory total (MB): 14584.45
	Swap memory used (MB): 2205.60
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Launched Version: 1.19.2-forge-43.2.0
	Backend library: LWJGL version 3.3.1 build 7
	Backend API: Intel(R) UHD Graphics GL version 3.2.0 - Build 30.0.100.9805, Intel
	Window size: 1920x1017
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	Graphics mode: fast
	Resource Packs: mod_resources, vanilla, file/FreshAnimations_v1.7.zip, file/VisibleOres2.8.zip
	Current Language: English (US)
	CPU: 4x Intel(R) Core(TM) i3-1005G1 CPU @ 1.20GHz
	OptiFine Version: OptiFine_1.19.2_HD_U_H9
	OptiFine Build: 20220820-230904
	Render Distance Chunks: 14
	Mipmaps: 0
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 3.2.0 - Build 30.0.100.9805
	OpenGlRenderer: Intel(R) UHD Graphics
	OpenGlVendor: Intel
	CpuCount: 4
	ModLauncher: 10.0.8+10.0.8+main.0ef7e830
	ModLauncher launch target: forgeclient
	ModLauncher naming: srg
	ModLauncher services: 
		mixin-0.8.5.jar mixin PLUGINSERVICE 
		eventbus-6.0.3.jar eventbus PLUGINSERVICE 
		fmlloader-1.19.2-43.2.0.jar slf4jfixer PLUGINSERVICE 
		fmlloader-1.19.2-43.2.0.jar object_holder_definalize PLUGINSERVICE 
		fmlloader-1.19.2-43.2.0.jar runtime_enum_extender PLUGINSERVICE 
		fmlloader-1.19.2-43.2.0.jar capability_token_subclass PLUGINSERVICE 
		accesstransformers-8.0.4.jar accesstransformer PLUGINSERVICE 
		fmlloader-1.19.2-43.2.0.jar runtimedistcleaner PLUGINSERVICE 
		modlauncher-10.0.8.jar mixin TRANSFORMATIONSERVICE 
		modlauncher-10.0.8.jar OptiFine TRANSFORMATIONSERVICE 
		modlauncher-10.0.8.jar fml TRANSFORMATIONSERVICE 
	FML Language Providers: 
		minecraft@1.0
		lowcodefml@null
		javafml@null
	Mod List: 
		client-1.19.2-20220805.130853-srg.jar             |Minecraft                     |minecraft                     |1.19.2              |DONE      |Manifest: a1:d4:5e:04:4f:d3:d6:e0:7b:37:97:cf:77:b0:de:ad:4a:47:ce:8c:96:49:5f:0a:cf:8c:ae:b2:6d:4b:8a:3f
		ftb-ultimine-forge-1902.3.5-build.65.jar          |FTB Ultimine                  |ftbultimine                   |1902.3.5-build.65   |DONE      |Manifest: NOSIGNATURE
		sophisticatedbackpacks-1.19.2-3.18.40.775.jar     |Sophisticated Backpacks       |sophisticatedbackpacks        |1.19.2-3.18.40.775  |DONE      |Manifest: NOSIGNATURE
		sophisticatedstorage-1.19.2-0.6.16.276.jar        |Sophisticated Storage         |sophisticatedstorage          |1.19.2-0.6.16.276   |DONE      |Manifest: NOSIGNATURE
		architectury-6.4.62-forge.jar                     |Architectury                  |architectury                  |6.4.62              |DONE      |Manifest: NOSIGNATURE
		ftb-library-forge-1902.3.11-build.166.jar         |FTB Library                   |ftblibrary                    |1902.3.11-build.166 |DONE      |Manifest: NOSIGNATURE
		balm-4.5.3.jar                                    |Balm                          |balm                          |4.5.3               |DONE      |Manifest: NOSIGNATURE
		jei-1.19.2-forge-11.5.0.297.jar                   |Just Enough Items             |jei                           |11.5.0.297          |DONE      |Manifest: NOSIGNATURE
		ftb-teams-forge-1902.2.11-build.87.jar            |FTB Teams                     |ftbteams                      |1902.2.11-build.87  |DONE      |Manifest: NOSIGNATURE
		ftb-chunks-forge-1902.3.14-build.218.jar          |FTB Chunks                    |ftbchunks                     |1902.3.14-build.218 |DONE      |Manifest: NOSIGNATURE
		forge-1.19.2-43.2.0-universal.jar                 |Forge                         |forge                         |43.2.0              |DONE      |Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
		sophisticatedcore-1.19.2-0.5.37.198.jar           |Sophisticated Core            |sophisticatedcore             |1.19.2-0.5.37.198   |DONE      |Manifest: NOSIGNATURE
		waystones-forge-1.19-11.1.0.jar                   |Waystones                     |waystones                     |11.1.0              |DONE      |Manifest: NOSIGNATURE
		collective-1.19.2-5.25.jar                        |Collective                    |collective                    |5.25                |DONE      |Manifest: NOSIGNATURE
		journeymap-1.19.2-5.9.0-forge.jar                 |Journeymap                    |journeymap                    |5.9.0               |DONE      |Manifest: NOSIGNATURE
	Crash Report UUID: 4f7b9042-2dc5-4c5d-a8d5-bbe4455260ed
	FML: 43.2
	Forge: net.minecraftforge:43.2.0

I am deseperatly looking for a fix I have spend a lot of time in that game 

Thank's for your reading and for the time spend helping me I'm clueless and also forgive my bad english I tried my best !

ps : I love your mod backpack include

Darkblaster14.

**Comment by P3pp3rF1y** [comment]
Optifine is the answer, they remove this method from forge and that's why you crash. So just removing optifine will fix it for you.

**Comment by Darkblaster14** [comment]
> 

Thank's a lot for your repply ! You've made my day it worked ! Did you know if replacing optifine by Ribidium whould cause the same issue ?

**Comment by zinerith** [comment]
Alternatively you could go to the sophisticatedcore config in your serverconfig folder (found in the world folder) and disable all the limited barrels to prevent this. Not an ideal solution, but it is an alternative one depending on your preference.

**Comment by P3pp3rF1y** [comment]
@zinerith why that reply? all barrels will crash so this is not going to help if players use regular barrels and the issue is really on optifine side which breaks mutliple mods and they don't seem interested in fixing it. So removing optifine and using a different performance mod like Rubidium instead is the right move here.

**Comment by zinerith** [comment]
Sorry, deleted my comment cuz I realized I was wrong. Went and tested it further with optifine, I can't find a single crash issue with the barrels and optifine in 1.19.2, whether limited or normal, upgraded or with upgrades in them. Not sure what is going on for OP, or if it's something optifine fixed? 

I had previously suggested to OP that they should disable limited barrels to fix it, but that was because I've been using the mod with optifine and having no issues, though I haven't used limited barrels. I still can't reproduce the crash though with the limited barrels now that I did try that. 

Optifine was recently updated before I was doing this, on 6/23 looks like, so I wonder if that fixed it and if OP would still have that issue on that version? the changelog is here if that helps: https://www.optifine.net/changelog?f=OptiFine_1.19.2_HD_U_I2.jar
